sim/warrior/items.go

```diff
@@ -99,10 +99,14 @@ var ItemSetMoltenGiantWarplate = core.NewItemSet(core.ItemSet{
 		2: func(agent core.Agent) {
 			character := agent.GetCharacter()
 			actionID := core.ActionID{SpellID: 99233}
+
+			warrior := agent.(WarriorAgent).GetWarrior()
+			talentReduction := time.Duration(warrior.Talents.BoomingVoice*3) * time.Second
+
 			buff := character.RegisterAura(core.Aura{
 				Label:    "Burning Rage",
 				ActionID: actionID,
-				Duration: 12 * time.Second,
+				Duration: 12*time.Second - talentReduction,
```

Comment:
Is this correct? You are reducing the duration of a buff based on number of points in Booming Voice talent. Shouldnt this be increasing it?